cypress/constants/common.ts

```diff
@@ -0,0 +1,7 @@
+export const DATA_SERVICES = 'Data Services'; // nav menu category option
+export const DATA_FEDERATION = 'Data Federation'; // nav menu sub category option
+export const STATUS_CARD_TITLE = 'Data Federation service';
```

Comment:
I can see we already have a constant file... either move this there or move that here as well...

Reply:
I think this is the constant file that I added. except this we don't have any

Reply:
/cypress/consts.ts